src/components/gallery/gallery.hbs

```diff
@@ -7,7 +7,11 @@
     <div class="gallery__main-image__zoom">
       <button data-modal-show="gallery-modal" aria-label="Enlarge the Image" class="btn btn--secondary ion-ios-expand"><span class="sr-only">enlarge</span></button>
       <div class="gallery__main-image__zoom-group">
-        <span class="gallery__zoom-icon ion-ios-search"></span>
+        <span class="gallery__zoom-icon ion-ios-search">        
```

Comment:
Not a blocker, but for future reference, we can clean out the `ion-ios-xxxx` type classes. They were used previously for the font-icon solution.